.version

```diff
@@ -1 +1 @@
-1.29.0
+2.00.0
```

Comment:
A few things:
1. This should be `2.0.0`.
2. If we want to v2 the SLS application version then we will need to do the v2 go module stuff. So SLS can still be imported as a go module. I've been thinking for a little bit that the type/struct information in the sls-common package should be split off into its own library to it can be versioned independently and the version of the service can be whatever it needs to be. 
3. Since we didn't rev the rest API from v1 to v2 for this change, I could also argue that we don't need to rev the application version 1.Y.Z to 2.Y.Z.

Reply:
@nieuwsma what do you think?

Reply:
I think the .version needs to go from 1xx to 2.0.0; We are leaving the API version (/v1) alone.
I agree the go module stuff will have to happen, but its necessary.

Reply:
I've updated the following
1. Changed 2.00.0 to 2.0.0
2. Added v2 to SLS's go module name